app/src/main/java/com/hedvig/app/feature/home/data/HomeRepository.kt

```diff
@@ -12,7 +12,7 @@ class HomeRepository(
     private val apolloClient: ApolloClient,
     private val localeManager: LocaleManager,
 ) {
-    fun home() = apolloClient
+    fun homeQueryFlow() = apolloClient
```

Comment:
I'm personally not a fan of what's called the hungarian notation (https://en.wikipedia.org/wiki/Hungarian_notation), i.e. including type information in the function/variable name. What are your thoughts on the subject? 🙂

Reply:
I agree, it's just that `homeQuery` was already taken and just `home()` really messed up my understanding whenever I saw it and didn't know what it was doing so I felt the urge to change it into anything 😅
This was resolved in a future commit anyway now.